ngx-tools/tsconfig.lint.json

```diff
@@ -7,5 +7,14 @@
     "dist_package",
     "e2e",
     "demo"
-  ]
+  ],
+  "compilerOptions": {
+    "paths": {
+      "@terminus/ngx-tools": ["./public-api.ts"],
+      "@terminus/ngx-tools/coercion": ["./coercion/public-api.ts"],
+      "@terminus/ngx-tools/keycodes": ["./keycodes/public-api.ts"],
+      "@terminus/ngx-tools/regex": ["./regex/public-api.ts"],
+      "@terminus/ngx-tools/testing": ["./testing/public-api.ts"]
+    }
+  }
```

Comment:
Without this, does it all roll up to the top level? and i guess it's hard to do tree shaken if that's the case?

Reply:
We have some independent packages that depend on each other, but during build, it does not like that you're including some code outside of it's package (for instance, `testing` depends on `keycodes`).  So, in those instances, we need to import from the top level instead of a relative path.  

Adding these tells the linter and specs to chill out on saying that path doesn't exist.